src/auth.ts

```diff
@@ -0,0 +1,45 @@
+/*
+auth.ts - Provides auth related methods for the Bluebutton class
+*/
+
+import BlueButton from "./index";
```

Comment:
You can actually just write this as: `import BlueButton from ".";`. Basically `index` is saying that is in the "default" thing to import from that directory

Reply:
I've also recently taken to letting my IDE do all the imports for me (and it uses the format I pasted above ^)